guides/rails_guides/markdown.rb

```diff
@@ -82,6 +84,11 @@ def generate_header
         @header = engine.render(@raw_header).html_safe
       end
 
+      def generate_description
+        sanitizer = Rails::Html::FullSanitizer.new
+        @description = sanitizer.sanitize(@header).squish
```

Comment:
How about adding `.presence` in the line instead of in the layout template?
```suggestion
        @description = sanitizer.sanitize(@header).squish.presence
```

Reply:
Good idea, but the problem is the file `index.html.erb` is rendered by ActionView Renderer(not `markdown.rb` renderer). So, `@description` isn't set when the `index.html` is rendered (As a result, the `description` becomes an empty string). 

My another suggestion is setting `description` variable in the view like existing `page_title` and `header_section `.

https://github.com/rails/rails/blob/86517942e469193e8624d5078d718785552c1270/guides/source/index.html.erb#L1-L9

Reply:
Yeah, looks good to me to treat the `description` like as the existing `page_title`.

Reply:
@kamipo Fixed! I added `description` to index.html and remove `presence` because all pages should have own description.

please re-review. :)